actionsystemtest/lib/action_system_test.rb

```diff
@@ -0,0 +1,67 @@
+# System tests are similar to Integration tests in that they incorporate multiple
+# controllers and actions, but can be used to simulate a real user experience.
```

Comment:
Let's get clearer on the value. System tests let you test JavaScript applications because they use a real browser as the testing agent. That's the difference and motivation. If you don't need to test JavaScript, don't bother with system tests, use integration tests.